src/Core/Services/Fido2ClientService.cs

```diff
@@ -33,7 +33,7 @@ public class Fido2ClientService : IFido2ClientService
             _makeCredentialUserInterface = makeCredentialUserInterface;
         }
 
-        public async Task<Fido2ClientCreateCredentialResult> CreateCredentialAsync(Fido2ClientCreateCredentialParams createCredentialParams)
+        public async Task<Fido2ClientCreateCredentialResult> CreateCredentialAsync(Fido2ClientCreateCredentialParams createCredentialParams, byte[] clientDataHash = null)
```

Comment:
🤔 Is there something on the Spec that doesn't allow us to put the `clientDataHash` in `Fido2ClientCreateCredentialParams`? Or you put it as an additional parameter here for another reason?

Reply:
No nothing like that. The `CreateCredentialParams` are the "tradititonal parameters" defined by WebAuthn, while `clientDataHash` is our own construction, because according to WebAuthn it's intended to be created by the `Fido2Client`. Honestly I just copied `passkey-rs` :D